pkg/apis/kops/networking.go

```diff
@@ -313,6 +316,9 @@ type CiliumNetworkingSpec struct {
 	// "never": Cilium allows all traffic regardless of policies in place.
 	// If unspecified, "default" policy mode will be used.
 	EnablePolicy string `json:"enablePolicy,omitempty"`
+	// EnableEndpointHealthChecking enables connectivity health checking between virtual endpoints.
+	// Default: true
+	EnableEndpointHealthChecking bool `json:"enableEndpointHealthChecking,omitempty"`
```

Comment:
How can default be true if this one is not a pointer?

Reply:
true ^^ switched to pointer